app/commands/v2/unpublish.rb

```diff
@@ -132,7 +133,23 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
+        scope = filter.filter(locale: locale, state: allowed_states)
+
+        Queries::GetLatest.call(scope).first
+      end
+
+      def supersede_previous_unpublished_items(content_item)
+        items = ContentItemFilter.similar_to(
+          content_item,
+          state: "unpublished",
+          base_path: nil,
+          user_version: nil,
+        )
+
+        items.each do |item|
+          next if item == content_item
+          State.supersede(item)
+        end
```

Comment:
I would prefer if this just found the previous unpublished, or published item, and superseeded it, as this mimics the behaviour in the publish endpoint (apart from I think there is a bug there also... in that it does not superseed previously unpublished content items).

If existing content item states are migrated, this should have the same behaviour, but the code will be simpler.